src/main/resources/application.conf

```diff
@@ -1,104 +1 @@
-      ]
-
-      bootstrap-nodes-scan-interval = 2 minutes
-    }
-
-    peer {
-      connect-retry-delay = 20 seconds
-      connect-max-retries = 3
-      disconnect-poison-pill-timeout = 5 seconds
-      wait-for-status-timeout = 30 seconds
-      wait-for-chain-check-timeout = 15 seconds
-
-      max-blocks-headers-per-message = 200
-      max-blocks-bodies-per-message = 200
-      max-receipts-per-message = 200
-      max-mpt-components-per-message = 400
-    }
-
-    rpc {
-      enabled: true
-      interface: "127.0.0.1"
-      port: 8546
-    }
-  }
-
-  blockchain {
-    genesis-difficulty = 17179869184
-    genesis-hash = "d4e56740f876aef8c010b86a40d5f56745a118d0906a34e69aec8c0db1cb8fa3"
-
-    // Doc: https://blog.ethereum.org/2016/07/20/hard-fork-completed/
-    dao-fork-block-number = "1920000"
-    dao-fork-block-total-difficulty = "39490964433395682584"
-    dao-fork-block-hash = "94365e3a8c0b35089c1d1195081fe7489b528a84b22199c916180db8b28ade7f"
-
-    chain-id = "3d"
-  }
-
-  fast-sync {
-    do-fast-sync = true
-    peers-scan-interval = 3.seconds
-    blacklist-duration = 30.seconds
-    start-retry-interval = 5.seconds
-    sync-retry-interval = 5.seconds
-    peer-response-timeout = 10.seconds
-    print-status-interval = 2.seconds
-    persist-state-snapshot-interval = 1.minute
-
-    max-concurrent-requests = 50
-    block-headers-per-request = 2048
-    block-bodies-per-request = 128
-    receipts-per-request = 128
-    nodes-per-request = 1000
-    min-peers-to-choose-target-block = 2
-    target-block-offset = 500
-
-    check-for-new-block-interval = 1.seconds
-    block-resolving-depth = 20
-  }
-
-  db {
-    iodb {
-      path = ${etc-client.datadir}"iodb/"
-    }
-    leveldb {
-      path = ${etc-client.datadir}"leveldb/"
-      create-if-missing = true
-      paranoid-checks = true // raise an error as soon as it detects an internal corruption
-      verify-checksums = true // force checksum verification of all data that is read from the file system on behalf of a particular read
-      cache-size = 0
-    }
-  }
-
-}
-
-akka {
-  loggers = ["akka.event.slf4j.Slf4jLogger"]
-  loglevel = "DEBUG"
-  logging-filter = "akka.event.slf4j.Slf4jLoggingFilter"
-  logger-startup-timeout = 30s
-}
+include "resources.conf"
```

Comment:
Actually I think the correct name for that file might be 'reference.conf'... not resources.conf.